app/controllers/resources_controller.rb

```diff
@@ -0,0 +1,26 @@
+class ResourcesController < ApplicationController
+  def index
+    @vets_phoenix = vets_phoenix
+    @vets_tucson = vets_tucson
+    @vets_other_areas = vets_other_areas
+    @pharmacy = pharmacy
+  end
+
+  private
+
+  def vets_phoenix
+    YAML.load_file('config/resources/vets_phoenix.yml')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.